tools/setup_helpers/cuda.py

```diff
@@ -5,6 +5,9 @@
 
 from .env import check_env_flag
 
+LINUX_HOME = '/usr/local/cuda'
+WINDOWS_HOME = 'C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v8.0'
```

Comment:
Do we need to specify the cuda version here? Doesn't CUDA install adds symlinks on windows?

Reply:
No, symlinks are rarely used in Windows. The other option may be execute a query in the registry.

Reply:
Yes, this seems like a very fragile hack. I don't know how the second option works, can you please explain that?

Reply:
The second option can be found using `where nvcc.exe` in Windows, just like what `which nvcc` does in Unix.

Reply:
Could this be found using 
`import os
WINDOWS_HOME = os.environ["CUDA_PATH"]`

Reply:
This is just a default value. You can find CUDA_PATH right down in the code.

Reply:
Still, can we make it search for the "vX.Y" pattern at the end? The problem with current code is that it assumes CUDA 8.0